src/utils/UrlUtils.ts

```diff
@@ -6,7 +6,7 @@
  */
 export class UrlUtils {
     public static readParams(url: string, responseMode: "query" | "fragment" = "query"): URLSearchParams {
-        if (!url) throw "Invalid URL";
+        if (!url) throw new Error("Invalid URL");
```

Comment:
thanks, i updated my previous comment, maybe i should have made a second one. What i edited:
Its probably better to use `TypeError` instead of `Error`, as this is what `new URL` would do. Sorry for the trouble. Should be easy to fix :-)